app/controllers/sessions_controller.rb

```diff
@@ -19,4 +19,14 @@ def destroy
     flash[:notice] = "Successfully logged out"
     redirect_to root_path
   end
+
+  private
+
+  def check_admin
```

Comment:
@erose357 Love it! Well done to checkout if the current user is an admin or not.